Flow.Launcher.Infrastructure/StringMatcher.cs

```diff
@@ -204,11 +187,16 @@ public MatchResult FuzzyMatch(string query, string stringToCompare, MatchOption
                 }
             }
 
-            // return acronym Match if possible
-            if (acronymMatchData.Count == query.Length && acronymScore >= (int) UserSettingSearchPrecision)
+            // return acronym match if all query char matched
+            if (acronymsMatched > 0 && acronymsMatched == query.Length)
             {
-                acronymMatchData = acronymMatchData.Select(x => translationMapping?.MapToOriginalIndex(x) ?? x).Distinct().ToList();
-                return new MatchResult(true, UserSettingSearchPrecision, acronymMatchData, acronymScore);
+                int acronymScore = (int)(acronymsMatched / acronymsTotalCount * 100);
```

Comment:
I believe since we are converting score to int, no need to use decimal, but just use an int for scoring will be fine. Using `acronymsMatched * 100 / acronymsTotalCount` should return the same result.

Reply:
cant use int here because you have decimals

Reply:
Well I mean since `(int)` do discard decimal, why do we need the decimal? We can simply `* 100` before division, and the result will be same, and calculation should be cheaper, though not a lot.

Reply:
oh gotcha